code/modules/mob/living/simple_animal/guardian/types/ranged.dm

```diff
@@ -98,8 +100,11 @@
 
 
 /obj/item/effect/snare/Crossed(AM as mob|obj)
```

Comment:
I think you're better off subtyping snare and overriding crossed for the surveillance traps